core/src/main/scala/filodb.core/metadata/Column.scala

```diff
@@ -27,6 +28,11 @@ trait Column {
   // More type safe than just using ==, if we ever change the type of ColumnId
   // TODO(velvia): remove this and just use id
   def hasId(_id: ColumnId): Boolean = name == _id
+
+  def toStringNotation: String = {
```

Comment:
This method is really not necessary, it duplicates code from DataColumn.toString which should be used instead.  Just make Dataset column definitions all DataColumns and you'll be done.

Reply:
The `toStringNotation` is used serialization and deserialization from/to `Config`. I considered using `toString` but that was one direction only. Deserialization was not possible really. At the same time, I didnt want to break any toString invocations happening today.

If you think it is safe to consolidate the two and replace `toString` with implementation of `toStringNotation` I can do so.

Reply:
When does one need to write out Config from a Column?